### DISCUSSION TITLE
Github Student Pack Activation

### DISCUSSION THREAD

**Discussion by ridafatima90** [community/community#53384] [GitHub Education]
Hello:

I have been applying for github student pack as i have to learn programming. As a proof of document i have attached my school issued id which mentions Session on it as well. My application is rejected multiple times without mentioning any concrete reason. Could anyone please assist me on that? I really need that to pace up my learning.

Thank You

**Comment by Ntshangase** [comment]
Hello @ridafatima90 

Sharing the rejection reason even though it's not concrete is helpful for you to be assisted, as it gives us more insight to your problem.

Maybe changing the type of document you are submitting can yield positive results, have you tried using your camera to capture your school issued ID, or submitting a proof of registration from your school.

Also make sure your GitHub has your correct name as GitHub does cross reference the documents  you submit with your GitHub account, If you have updated your GitHub account, It's recommended that you Logout then Login to your GitHub account then proceed to re-apply.

**Reply by ridafatima90** [reply]
Thanks for reply. I have uploaded school issued dated id which mention the period of enrollment. Unfortunately, my school doesn't issue school email id. Hence, I can't provide that. Kindly assist. Thanks

**Reply by Ntshangase** [reply]
If you don't have a school email that's still fine GitHub will still approve your application as long as the documentation submitted demonstrates that you are a currently registered student.

Just make sure your documents are clear  i.e. https://github.com/orgs/community/discussions/51831

**Reply by ridafatima90** [reply]
Is there anything else required? Could you please check!

**Comment by ridafatima90** [comment]
Done. Sent to you via registered email!

**Comment by ridafatima90** [comment]
Dear All:

Please find a soft copy of my id as well attached herewith.

**Comment by ridafatima90** [comment]
Hello:
I have again reapplied with new document as the reason mentioned that uploaded document was previously used during my application.
Kindly check please.

Thank you
Regards

**Reply by Ntshangase** [reply]
Hello

I do not have access to the back-end i was just trying to assist you and help get your application to be accepted as i have went through the similar application and i know some common problems that are usually faced when applying.

After you apply posting a screenshot of your rejection reason can help the community assist you better.

**Reply by ridafatima90** [reply]
Please find the attached screenshot.
![1](https://user-images.githubusercontent.com/128382099/234319296-0329cdc7-d03b-467d-8de8-9efa8fcf9980.JPG)

**Comment by ridafatima90** [comment]
Hi,
Please find the attached document.

**Comment by Ntshangase** [comment]
@samus-aran I think this is a robot.

**Reply by ridafatima90** [reply]
![1](https://user-images.githubusercontent.com/128382099/234612894-73dadd1a-8ccc-40a0-abb3-98a4d8881dc4.JPG)
This is the sixth attempt. Applied again with fresh document.

**Reply by samus-aran** [reply]
Thanks @Ntshangase for the mention. 👍

**Reply by ridafatima90** [reply]
@all 7th one!
![7](https://user-images.githubusercontent.com/128382099/234920173-f536255e-cd3f-42fe-829a-6d0efc1bc646.JPG)

**Reply by ridafatima90** [reply]
Can anyone assist me on that? Now the UI is displaying different kind of message displaying there is an issue with my github account. Thanks!
![2](https://user-images.githubusercontent.com/128382099/234922071-15a66329-c69b-4feb-9a34-5a3dac03c854.JPG)

**Reply by ridafatima90** [reply]
Dear @All:
I am posting here all rejection messages which i got from starting on 20th of March 2023 sequentially. I did not get any concrete message stating why it is being rejected?. Now the situation is i cannot even apply for that as the message(outstanding issue with github account) is being displayed. Can anyone help me on that.? Thank you!
![1](https://user-images.githubusercontent.com/128382099/234927397-25803195-e95e-4d69-9c77-e3dd4599f152.JPG)
![2](https://user-images.githubusercontent.com/128382099/234927442-24d03993-f12f-4a2a-9954-02be295a198d.JPG)
![3](https://user-images.githubusercontent.com/128382099/234927486-f84261a4-13b2-4a1c-8f0f-0e569e19f103.JPG)
![4](https://user-images.githubusercontent.com/128382099/234927582-29c71c02-9a17-4804-a260-71f52e7f14b3.JPG)
![5](https://user-images.githubusercontent.com/128382099/234927620-3f1b142c-3dba-436b-80f6-566d2d389e08.JPG)
![6](https://user-images.githubusercontent.com/128382099/234927648-38186298-1369-440b-9960-e1aed485633d.JPG)
![7](https://user-images.githubusercontent.com/128382099/234927677-d92df29a-c3d8-46a4-9f99-841a423588f0.JPG)

**Reply by samus-aran** [reply]
Hi @ridafatima90 

This message can be an indication of many things that only the Support team can answer and resolve. So please do contact [Account or billing issues](https://support.github.com/request) Support and they will explain the details of this message.

I will be locking this thread as I feel it has ran it's course. Once you have resolved your issue with Support on the above message you can create a new discussion to help you with your application.

**Comment by snasif** [comment]
I'm having the same issues, I don't how else to find help. My reason for being rejected are:

You are unlikely to be verified until you have completed your GitHub user profile with your full name as it appears in your academic affiliation document plus a short bio.

You appear not to be near any campus location for the school you have selected. If you are a distance learner then your school-provided academic affiliation documentation must state so.

I'm an online student and I've updated my profile mulitple times. Not sure what else to do.

**Reply by ridafatima90** [reply]
I am also clueless. This is my 7th attempt
